py/torch_tensorrt/dynamo/lowering/passes/__init__.py

```diff
@@ -0,0 +1,78 @@
+import logging
+from functools import wraps
+from typing import Callable, Optional, Tuple
+
+import torch
+
+# Import and order lowering passes and pass manager
+from .constant_folding import constant_fold
+from .pass_manager import DynamoPassManager
+from .repair_input_as_output import repair_input_as_output
+
+ATEN_LOWERING_PASSES = DynamoPassManager.build_from_passlist(
```

Comment:
Lets minimize code in __init__.py files, Lets create a `dynamo.lowering.passes._aten_lowering_pass` and import from there